vtex-apps/menu
# Feature/optimize rendering from settings

#### What problem is this solving?
Allows enabling submenu rendering optimization on store settings.

Works in tandem with https://github.com/vtex-apps/store/pull/483



#### How to test it?
The admin setting can be seen on https://lbebbermenu1--carrefourbr.myvtex.com/admin/cms/store

![Screen Shot 2020-09-03 at 22 28 53](https://user-images.githubusercontent.com/5691711/92189375-deafe180-ee34-11ea-8302-0b173157d646.png)

To see the feature itself in action, go to https://www.carrefour.com.br/?ak-testab=vtex&workspace=lbebbermenu1&v=01, open devtools, and on the "Elements" panel, search for "Geladeiras Samsung". The search might hit a JSON metadata block, but no element containing that text should appear.

Contrast with https://www.carrefour.com.br/?ak-testab=vtex, where searching for the same string will encounter an invisible submenu tag containing that text, which is a submenu item from the header menu.



[Workspace](Link goes here!)

#### Screenshots or example usage:



#### Describe alternatives you've considered, if any.



#### Related to / Depends on



#### How does this PR make you feel? [:link:](http://giphy.com/)



![](put .gif link here - can be found under "advanced" on giphy)
